encoders/nlp/TransformerTorchEncoder/tests/test_transformertorchencoder.py

```
@@ -111,3 +117,10 @@ def test_parameter_override(encoder):
     assert encoder.pretrained_model_name_or_path == encoder_preset['pretrained_model_name_or_path']
     assert encoder.pooling_strategy == encoder_preset['pooling_strategy']
     assert encoder.model_save_path == encoder_preset['model_save_path']
+
+@pytest.mark.parametrize('encoder', encoders_parameters[0:1], indirect=['encoder'])
```

can we test also a different layer than -1 and see that it indeed works? And also test what happens if we use -100 as an edgecase?

---

Please check the the tests. I have added a tests for layer `-2`, as well as a test for layer `100` (which should raise an error)

---

Sorry, or did you mean you want to also test -100, as well as 100?

---

ah yes sorry did not realize the test. It would be cool to test -100 although it may be overkilling. What is really important is to increase the versions in manifest.yml

---

I have done that now :)

---

where is the test?

---

ah I see you changed the range, can there be a test with large negative value? It is good also as documentation to know edge cases behavior

---

Ok, added test for -100 as well